makeflow/src/makeflow.c

```diff
@@ -874,7 +880,14 @@ static void makeflow_run( struct dag *d )
 	struct dag_node *n;
 	batch_job_id_t jobid;
 	struct batch_job_info info;
+        timestamp_t last_time = timestamp_get();
+        timestamp_t start = timestamp_get();
+        int first_report = 1;
 
+        //reporting to catalog
+        if(in_n_mode){
```

Comment:
It's not the wording, just that "N switch" or "N mode" doesn't relate to anything in the code beyond the option parsing. If the `-N` flag controls two things, it might be clearer to have two variables, even if they're simply set together